app/src/main/AndroidManifest.xml

```diff
@@ -66,6 +67,11 @@
             android:name="com.google.android.geo.API_KEY"
             android:value="ADD_YOUR_OWN_GEO_API_KEY" />
 
+        <!-- Bugsnag API_KEY for Dimelo -->
+        <meta-data
+            android:name="com.bugsnag.android.API_KEY"
+            android:value="5f7160ae904d332495d1899e0da8955e" />
```

Comment:
Dunno how I feel about having our bugsnag api_key on a public repo which makes it possible for anyone to push their crap into our bugsnag :(, any opinon @jarthod ?

Though I don't know how we would've done that without having it here as being asked for the api_key each time someone wants to compile the app would be painful in the long run :/

Reply:
Is this API key only for the sample app or for the whole bugsnag account? Indeed it would be a bit better if the key was in a file not committed but if it's only for this app It's acceptable IMO as we can revoke it when we want.

Reply:
I just checked and this API key is for the Sample-app project so I guess this is fine :+1:

Reply:
yes i created a new project for that (dimelo-sampleapp)